data.json

```diff
@@ -970,5 +970,11 @@
     "githubUsername": "adityamangal1",
     "imageUrl": "https://user-images.githubusercontent.com/68494604/120438793-6bc1a000-c39f-11eb-845b-c145db5e2f9b.png",
     "issueNumber": 544
+ },
+ {
```

Comment:
@adityamangal1 May I know why you added @skjha1 details 🤔

Reply:
Oops. I also notice. the imageUrl is wrong.

Reply:
I think the image URL is right can you recheck it?

Reply:
> I think the image URL is right can you recheck it? @vinzvinci

your info is correct but remove the @skjha1 information 
do you need any help?

Reply:
Yes I need some help @deephunt3r

Reply:
Yeah! What can I do for you

Reply:
Okay i will re do the process then